editor/animation_track_editor.h

```diff
@@ -648,12 +651,17 @@ class AnimationTrackEditor : public VBoxContainer {
 	void _add_track(int p_type);
 	void _new_track_node_selected(NodePath p_path);
 	void _new_track_property_selected(const String &p_name);
-
+	void _move_track_to_node(NodePath p_path, int p_track);
+	void _change_track_property_selected(const String &p_name, int p_track);
+	void _change_track_target_node_pressed(int p_track);
+	void _change_track_target_property_pressed(int p_track);
```

Comment:
The order of method definitions in cpp file should match the order of declarations (currently it doesn't).